src/CalcViewModel/DataLoaders/CurrencyDataLoader.cpp

```diff
@@ -547,9 +548,15 @@ bool CurrencyDataLoader::TryParseStaticData(_In_ String^ rawJson, _Inout_ vector
     }
 
     // TODO - MSFT 8533667: this sort will be replaced by a WinRT call to sort localized strings
-    sort(begin(staticData), end(staticData), [](CurrencyStaticData unit1, CurrencyStaticData unit2)
+
+    auto loc = std::locale("");  //Use the user-preferred locale to sort country names
```

Comment:
So, I'm not sure if this would pick on "alternate sorts" that the user might have selected in the Settings/Regional Control Panel.

Reply:
If it was the case, it would be even better, no?

Reply:
No, it would mean that the user's selection in the Settings was being disregarded, which is a bad thing.

Though on second though if that was the case, then it feels like it should be bug on the library or CRT.

Reply:
> No, it would mean that the user's selection in the Settings was being disregarded, which is a bad thing.

We are on the same page in fact, my answer was perhaps ambiguous, "if the applications uses the alternate sort" selected by the user, it would be even better" (Not tested but it's probably already the case.)